### PR TITLE
Saws

### PR DESCRIPTION
**Pull request by agcopenhaver**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/saws/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5f878a6996a7d6a9bf7a7617baa7ed1a1a9249f803cbc00874ad8c74ff82b639
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (marcelotrevisani) [inline recipes/saws/meta.yaml]: Please, add ```noarch: python```

File: recipes/saws/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+
```
Comment (marcelotrevisani) [inline recipes/saws/meta.yaml]: There are two empty new lines here, could you please remove one?

File: recipes/saws/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+requirements:
+  build:
```
Comment (marcelotrevisani) [inline recipes/saws/meta.yaml]: Could you please remove the empty build section?

**Comment by agcopenhaver** [comment]
I don't think this will work for noarch because it contains binary scripts

**Comment by marcelotrevisani** [comment]
> I don't think this will work for noarch because it contains binary scripts

Ah ok then, I thought that was a pure python project. Please remove it then

**Pull request merged**